src/main.test.js

```diff
@@ -169,3 +169,25 @@ test("Wrong Syntax for default value,missing value", () => {
     message: "EnvSpecSyntaxError: Expected default value after ="
   });
 });
+
+test("Test for serialization of data", () => {
+  const myForm = document.createElement("form");
```

Comment:
Can you add also a `<select><option>...` input as well to check that restricted choice input get serialized as well?